src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -204,12 +259,22 @@ public void run() {
             }
         }
 
-        long checkDiskUsage() {
+        long checkDiskUsage(Map<FileStore, Double> volumePercents) {
             return logDirs.stream()
                 .filter(Files::exists)
                 .map(path -> apply(() -> Files.getFileStore(path)))
                 .distinct()
-                .mapToLong(store -> apply(() -> store.getTotalSpace() - store.getUsableSpace()))
```

Comment:
I think you can use `apply` here, but if not then let's just get rid of it entirely and use try/catch in the lambda above.